Examples/Along-slope-velocities.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>Anything we can do about these RuntimeWarnings? Maybe at the very least we could add a comment about why they are there and that they are ok to ignore?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/420/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p># warnings are for dividing by NaN/0,&nbsp;</p><p># i.e. when there is no topographic gradient and warning can be ignored</p>